stores/postgres/car_sql.go

```diff
@@ -0,0 +1,27 @@
+package postgres
+
+const (
+	carsCreateTable = `
+	CREATE TABLE IF NOT EXISTS cars (
+		id varchar(20) primary key,
+		make varchar(128) NOT NULL,
+		model varchar(128),
+		year int NOT NULL ,
+		color varchar(128) NOT NULL,
+		user_id varchar(20) NOT NULL,
+		created_at timestamptz DEFAULT NOW(),
+		updated_at timestamptz DEFAULT NOW()
+	);`
+
+	carsGetAllSQL = "SELECT * FROM cars WHERE id >= $1 LIMIT $2"
```

Comment:
Doing id > $1 makes more sense, becasue lastID is the last ID from the previous page so they already have this entity